packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/documentservice/DocumentResource.java

```diff
@@ -60,11 +58,12 @@ public Response getDocuments(
 		@QueryParam("additional_fields") String additional_fields,
 		@QueryParam("match") String match,
 		@QueryParam("known_entities") String known_entities,
-		@QueryParam("github_url") String github_url
+		@QueryParam("github_url") String github_url,
+		@QueryParam("similar_to") String similar_to
 	) {
 
 		// only go ahead with the query if at least one param is present
-		if (docid != null || doi != null || term != null || github_url != null) {
+		if (docid != null || doi != null || term != null || github_url != null || similar_to != null) {
```

Comment:
Not for this PR but I have some strong feelings on how to cleanup and refactor this resource class as part of the next PR.  If you're doing back end work here let me know!